standard/league_icon.lua

```diff
@@ -0,0 +1,184 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:LeagueIcon
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local LeagueIcon = {}
+local Class = require('Module:Class')
+local Template = require('Module:Template')
+local Logic = require('Module:Logic')
+local String = require('Module:StringUtils')
+
+---display an image in LeagueIconSmall fashion
```

Comment:
It would be good to explain what "LeagueIconSmall fashion" means.

Reply:
adjusted the wording a bit

Reply:
Yea, that's still not clear, I'm looking for an explanation without using  `LeagueIconSmall`

Reply:
added 2 more lines explaining it